nixos/modules/programs/command-not-found/command-not-found.pl

```diff
@@ -7,6 +7,7 @@
 use Config;
 
 my $program = $ARGV[0];
+my $quoted_command = '"' . shell_quote(@ARGV) . '"';
```

Comment:
is this guaranteed to be correct?

Reply:
likely not - but for corner cases what is 'correct' might be different depending on what shell the user is using, and taking that into account seems excessive.